src/languages/nginx.coffee

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+
+  name: "Nginx"
+  namespace: "nginx"
+
+  ###
+  Supported Grammars
+  ###
+  grammars: [
+    "nginx"
+  ]
+
+  ###
+  Supported extensions
+  ###
+  extensions: [
+    "conf"
+  ]
+
+  defaultBeautifier: "Nginx Beautify"
+
+  options:
+    spaces:
```

Comment:
I try to keep the same names for the same options, such as `indent_size` and `indent_char`: https://github.com/Glavin001/atom-beautify/blob/master/src/languages/javascript.coffee#L35-L43

Then in the beautifier itself apply some transformations to the option values, like https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/prettydiff.coffee#L10-L44